src/features/category/ui/routes/CategoryRoute/CategoryRoute.tsx

```
@@ -0,0 +1,123 @@
+import { useCategoryByUid } from '@/features/category/api/useCategoryQuery';
+import Layout from '@/shared/ui/layout/Layout';
+import { cn } from '@/shared/ui/styles/utils';
+import SubPageHeader from '@/shared/ui/SubPageHeader';
+import UnderlinedTextInput from '@/shared/ui/UnderlinedTextInput';
+
+const CategoryRoute: React.FC = () => {
+  const router = useRouter();
+  const { category_uid } = useParams({ strict: false });
+  const uid = category_uid as string;
+
+  const { category, error } = useCategoryByUid(uid);
```

ESLint

[error] 'category' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)